doc/manuals/admin/extra/ha.md

```diff
@@ -1,5 +1,8 @@
 # Create an Orion Context Broker cluster in High Availability
 
+**WARNING**: this procedure is obsolete, as it is based in Orion installation using RPMs,
+which support was removed in Orion 3.7.0. It should be updated by its original author.
```

Comment:
@chicco785 if I remember correctly, you were the author of this ha.md file.

What do you think about this Warning message?

Reply:
I originally added this because it was requested by you :) for me we can remove it.

Reply:
> I originally added this because it was requested by you :) for me we can remove it.

Do you mean the ha.file itself? Not sure why this contribution was motivated (it was more than 4 years ago), but if you are no longer supporting this file and/or it has obsolete content that it's not going to be updated, I agree it's better to remove it.

Reply:
What do you think?

Reply:
Whichever solution, let's do it in a next PR.

NTC (in this PR)